lib/cppcheck.cpp

```diff
@@ -520,6 +527,9 @@ unsigned int CppCheck::checkClang(const FileWithDetails &file)
             fdump << "    <c version=\"" << mSettings.standards.getC() << "\"/>\n";
             fdump << "    <cpp version=\"" << mSettings.standards.getCPP() << "\"/>\n";
             fdump << "  </standards>\n";
+            const std::string libs = getLibraryDumpData();
```

Comment:
these 3 lines of code can be refactored into 1 line of code. It does not hurt to output an empty string.